spring-boot-project/spring-boot-docs/src/main/asciidoc/deployment.adoc

```diff
@@ -294,28 +298,30 @@ binaries instead, add the following to your `.elasticbeanstalk/config.yml` file:
 	deploy:
 		artifact: target/demo-0.0.1-SNAPSHOT.jar
 ----
+====
 
-
-
-====== Reduce costs by setting the environment type
-By default an Elastic Beanstalk environment is load balanced. The load balancer has a cost
-perspective, to avoid it, set the environment type to "`Single instance`" as described
-http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-create-wizard.html#environments-create-wizard-capacity[in the Amazon documentation].
-Single instance environments can be created using the CLI as well using the following
-command:
+[TIP]
+.Reduce costs by setting the environment type
+====
+By default an Elastic Beanstalk environment is load balanced. The load balancer has a
+significant cost. To avoid that cost, set the environment type to "`Single instance`", as
+described in
+http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/environments-create-wizard.html#environments-create-wizard-capacity[the Amazon documentation].
+You can also create single instance environments by using the CLI by using the
```

Comment:
"by using the CLI by using" -> "by using the CLI and"?

Reply:
Yes. Thanks.